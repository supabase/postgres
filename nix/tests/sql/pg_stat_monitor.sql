select
  *
from
  pg_stat_monitor
where
  false;
