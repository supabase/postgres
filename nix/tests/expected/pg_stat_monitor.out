select
  *
from
  pg_stat_monitor
where
  false;
 bucket | bucket_start_time | userid | datname | client_ip | queryid | toplevel | top_queryid | query | comments | planid | query_plan | top_query | application_name | relations | cmd_type | cmd_type_text | elevel | sqlcode | message | calls | total_exec_time | min_exec_time | max_exec_time | mean_exec_time | stddev_exec_time | rows_retrieved | plans_calls | total_plan_time | min_plan_time | max_plan_time | mean_plan_time | stddev_plan_time | shared_blks_hit | shared_blks_read | shared_blks_dirtied | shared_blks_written | local_blks_hit | local_blks_read | local_blks_dirtied | local_blks_written | temp_blks_read | temp_blks_written | blk_read_time | blk_write_time | resp_calls | cpu_user_time | cpu_sys_time | wal_records | wal_fpi | wal_bytes | state_code | state 
--------+-------------------+--------+---------+-----------+---------+----------+-------------+-------+----------+--------+------------+-----------+------------------+-----------+----------+---------------+--------+---------+---------+-------+-----------------+---------------+---------------+----------------+------------------+----------------+-------------+-----------------+---------------+---------------+----------------+------------------+-----------------+------------------+---------------------+---------------------+----------------+-----------------+--------------------+--------------------+----------------+-------------------+---------------+----------------+------------+---------------+--------------+-------------+---------+-----------+------------+-------
(0 rows)

